<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditConfigDialog</class>
 <widget class="QDialog" name="EditConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="toolWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="buttonUndo">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../ConfigWidgets.qrc">
          <normaloff>:/oxygen/edit-undo</normaloff>:/oxygen/edit-undo</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonRedo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../ConfigWidgets.qrc">
          <normaloff>:/oxygen/edit-redo</normaloff>:/oxygen/edit-redo</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonFind">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../ConfigWidgets.qrc">
          <normaloff>:/oxygen/edit-find</normaloff>:/oxygen/edit-find</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonBuilder">
        <property name="toolTip">
         <string>Toggle attribute builder</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../ConfigWidgets.qrc">
          <normaloff>:/builder/default_star.png</normaloff>:/builder/default_star.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="attributeBuilderParentWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="CCfg::Ui::AttributeBuilderWidget" name="attributeBuilder" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="CCfg::Ui::ClosableTabWidget" name="workspaceUi"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>splitter</zorder>
  <zorder>buttonBox</zorder>
  <zorder>toolWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CCfg::Ui::ClosableTabWidget</class>
   <extends>QTabWidget</extends>
   <header>CCfg/Ui/ClosableTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CCfg::Ui::AttributeBuilderWidget</class>
   <extends>QWidget</extends>
   <header>CCfg/Ui/AttributeBuilderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../ConfigWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
